inc/zoo/swar/SWAR.h

```diff
@@ -367,7 +367,10 @@ struct BooleanSWAR: SWAR<NBits, T> {
 };
 
 template <int NBits, typename T>
-BooleanSWAR(Literals_t<NBits, T>, const bool (&values)[BooleanSWAR<NBits, T>::Lanes]) -> BooleanSWAR<NBits, T>;
+BooleanSWAR(
+    Literals_t<NBits, T>,
+    const bool (&values)[BooleanSWAR<NBits, T>::Lanes])
+  -> BooleanSWAR<NBits, T>;
```

Comment:
Respecting 80 columns is of less importance than having a consistent way to express "detail".
Closing the list of arguments in this deduction guide far to the right breaks the context of "where does the list of arguments close?":
```
BooleanSWAR(
    Literals_t<NBits, T>,
    const bool (&values)[BooleanSWAR<NBits, T>::Lanes]
)
```
I don't understand your insistence in doing things in a different way.
This then goes to worsen the layout by introducing a two-space accidental indentation level.
Again: please desist.

Reply:
No dramas, will update. For your sanity I've updated my indentation level for this project specifically.

Reply:
You have said something similar three times.
Desist?
What about giving a chance to expressing something with whitespace? could that help you see that clang-format is made by people who don't see whitespace as a communication channel, for more people of the same?
What about learning about "tools that help you make the wrong thing easier", Brit Kevlin Henney?